### PR TITLE
Added param to Install Infra and workload nodes on cluster with worke…

### PR DESCRIPTION
**Pull request by SachinNinganure**
[OCPQE-9178]
(https://issues.redhat.com/browse/OCPQE-9178)

Install Infra and workload nodes on cluster with worker nodes <=50

TC Run: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/sninganu-e2e-benchmarking-multibranch-pipeline/job/cluster-workers-scaling/8/console

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -113,7 +114,7 @@ pipeline {
         '''
         }
         script{
-          if(params.WORKER_COUNT.toInteger() > 50) {
+	  if(params.WORKER_COUNT.toInteger() > 50 || params.infra_workload_install == true ) {
```
Comment (paigerube14) [inline Jenkinsfile:117]: INFRA_WORKLOAD_INSTALL should be capitalized

Comment (SachinNinganure) [reply Jenkinsfile]: I have consolidated/updated the suggested change and committed to my branch;

**Comment by paigerube14** [comment]
looks like there is an extra file you pushed called ocp-qe-perfscale-ci-2, can you try to delete that

**Comment by paigerube14** [comment]
/lgtm

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/sninganu-e2e-benchmarking-multibranch-pipeline/job/cluster-workers-scaling/10/console

**Pull request merged**